track_oracle/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ else()
 endif()
 
 find_package( TinyXML REQUIRED )
+find_package( vxl REQUIRED )
```

Comment:
Is this line needed? It looks like it should be doing the find inside kwiver-depends-VXL.cmake, which you've already included.

Reply:
Indeed, both that line and the find_package( TinyXML ) above it seem to be redundant... patch revision forthcoming...